content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+  Gateway Enhancement Proposals (GEPs) for features accepted into the release,
+  with emphasis on getting consensus around the design and graduation criteria
+  of the feature.
+
+- **API Refinement and Documentation** (3-5 weeks): contributors implement the
+  features in the Gateway API controllers and write the necessary
+  documentation.
+
+- **SIG-Network Review and Release Candidates** (2-4 weeks): maintainers get
+  the required upstream review, build release candidates, and release the new
+  version.
+
+Gateway API 1.2.0 is the first release to use the new process, and although
+there are the usual rough edges of anything new, we believe that it went well.
+To that end, we've just started the Scoping phase for Gateway API 1.3, which
+will end on November 19, 2024, with the release expected around the end of
+January 2025.
+
+### `gwctl` Moves Out
+
+The `gwctl` CLI tool has moved into its very own repository,
+https://github.com/kubernetes-sigs/gwctl. `gwctl` has proven a valuable tool
+for the Gateway API community; moving it into its own repository will, we
+believe, make it easier to maintain and develop. As always, we welcome
+contributions; `gwctl` makes quite a few things about Gateway API and
+Kubernetes easier to manage.
+
+### Maintainer Changes
+
+Rounding out our changes to the project itself, we're pleased to announce that
+[Mattia Lavacca] has joined the ranks of Gateway API Maintainers! We're also
+sad to announce that [Keith Mattix] has stepped down as a GAMMA lead --
+happily, [Mike Morris] has returned to the role. We're grateful for everything
+Keith has done, and excited to have Mattia and Mike on board.
+
+[Mattia Lavacca]: https://github.com/mlavacca
+[Keith Mattix]: https://github.com/@keithmattix
+[Mike Morris]: https://github.com/@mikemorris
+
+## Gateway API background
+
+The idea of Gateway API was initially [proposed](https://youtu.be/Ne9UJL6irXY?si=wgtC9w8PMB5ZHil2)
+at the 2019 KubeCon San Diego as the next generation
+of Ingress API. Since then, an incredible community has formed to develop what
+has likely become the
+[most collaborative API in Kubernetes history](https://www.youtube.com/watch?v=V3Vu_FWb4l4).
+Over 200 people have contributed to this API so far, and that number continues to grow.
+
+The maintainers would like to thank _everyone_ who's contributed to Gateway API, whether in the
+form of commits to the repo, discussion, ideas, or general support. We literally
+couldn't have gotten this far without the support of this dedicated and active
+community.
+
+## Try it out
+
+Unlike other Kubernetes APIs, you don't need to upgrade to the latest version of
+Kubernetes to get the latest version of Gateway API. As long as you're running
+Kubernetes 1.26 or later, you'll be able to get up and running with this
+version of Gateway API.
+
+To try out the API, follow our [Getting Started Guide](https://gateway-api.sigs.k8s.io/guides/).
```

Comment:
Don't forget to mention all the implementations that have submitted conformance reports for v1.2